### DISCUSSION TITLE
json to video

### DISCUSSION THREAD

**Discussion by erind7998** [community/community#61717] [Actions]
### Select Topic Area

Show & Tell

### Body

Need help turning this into a video
![llll](https://github.com/community/community/assets/136864277/64f060c8-8fae-4d30-a595-dbbcc875bdcb)

**Comment by nirav-gajera** [comment]
https://web.json2video.com/
https://shotstack.io/

or add frames to video by importing the images 
```

{
  "frames": [
    {
      "image": "image1.jpg",
      "duration": 1000
    },
    {
      "image": "image2.jpg",
      "duration": 1000
    },
    {
      "image": "image3.jpg",
      "duration": 1000
    }
  ]
}

```
